### PR TITLE
Ru guide translation

### PR DESCRIPTION
**Pull request by gazay**
I think this translation need to be reviewed. Also I'm installing latex to check that PDF generating fine from this. And some of topics I left untranslated as done in `de` version for example, as I'm beginner in Haskell and don't deep understand topics to translate it properly.

### CODE REVIEW

File: guide-ru.md [hunk 1]
```diff
@@ -0,0 +1,835 @@
...
+
+* [En Français](guide-fr.md)
+
+* [In Italiano](guide-it.md)
+
+* [Em Português](guide-pt.md)
+
+* [繁體中文](guide-zh_tw.md)
+
+#### *Не страдайте над одним топиком, который вы не можете понять сразу*. Двигайтесь дальше!
```
Comment (yaroslav) [inline guide-ru.md]: @gazay «Не мучайтесь»?

File: guide-ru.md [hunk 2]
```diff
@@ -0,0 +1,835 @@
...
+
+[Почтовые рассылки](https://wiki.haskell.org/Mailing_lists).
+
+
+### Политика сообщества
+
+[Смотрите пост про обучение от Криса Дона](http://chrisdone.com/posts/teaching)
+
+Будьте дружелюбны и вежливы. Грубость и хамство испугает собеседника
+и отталкнет от участия в общении.
```
Comment (yaroslav) [inline guide-ru.md]: оттолкнет

File: guide-ru.md [hunk 3]
```diff
@@ -0,0 +1,835 @@
...
+
+Слабая, неконструктивная критика удовлетворяет лишь
+критикующего, но не помогает критикуемому.
+
+Не описывайте вещи как "легкие" или "тривиальные".
+Вы заставляете людей чувствовать себя ужасно за то,
+что они усердно и много работают ради своего прогресса.
+Медленно-обучающиеся люди зачастую самые дотошные обучающиеся,
```
Comment (yaroslav) [inline guide-ru.md]: Медленные в обучении люди зачастую самые дотошные

File: guide-ru.md [hunk 4]
```diff
@@ -0,0 +1,835 @@
...
+этому наоборот надо радоваться!
+
+Не симулируйте удивление. Не изображайте удивление, когда кто-либо
+говорит, что он не знает что-то. Они будут чувствовать себя ужасно,
```
Comment (yaroslav) [inline guide-ru.md]: Он будет

File: guide-ru.md [hunk 5]
```diff
@@ -0,0 +1,835 @@
...
+трансофобия и другие виды фобий не приветствуются и не будут приняты терпимо.
+
+---
+
+[Правила хорошего тона от Recurse Center](https://www.recurse.com/manual). Спасибо за их публикацию Recurse Center.
+
+# Что такое Haskell, GHC и Cabal?
+
+Haskell - это язык программирования, изложенный в спецификациях,
+последняя опубликована в 2010 году.
```
Comment (yaroslav) [inline guide-ru.md]: последняя версия которых

File: guide-ru.md [hunk 6]
```diff
@@ -0,0 +1,835 @@
...
+Эта спецификация доступна как [онлайн-документ](http://www.haskell.org/onlinereport/haskell2010/).
+
+## GHC
+
+[GHC](http://www.haskell.org/ghc/) - это самый популярный способ работы с языком Haskell. Он включает в себя компилятор, REPL (интерпритатор), пакетный менеджер и другие полезные вещи.
+
+## Cabal
+
+[Cabal](https://www.haskell.org/cabal/download.html) осуществляет менеджмент проектов и разрешение зависимостей.
```
Comment (yaroslav) [inline guide-ru.md]: Cabal не может осуществлять менеджмент проектов :) Надо выбрать другую формулировку

File: guide-ru.md [hunk 7]
```diff
@@ -0,0 +1,835 @@
...
+Это способ установить проект, обычно в их песочницу (изолированное окружение).
```
Comment (yaroslav) [inline guide-ru.md]: в его песочницу / в «песочницу»

File: guide-ru.md [hunk 8]
```diff
@@ -0,0 +1,835 @@
...
+
+## НЕ УСТАНАВЛИВАЙТЕ HASKELL PLATFORM
+
+Вместо следования инструкциям на Haskell.org, смотри инструкции ниже, как установить GHC и Cabal.
+
+# Установка GHC и Cabal
+
+## Ubuntu
+
+[Этот PPA](http://launchpad.net/~hvr/+archive/ghc) лучший и именно его я использую на
```
Comment (yaroslav) [inline guide-ru.md]: лучший, и

File: guide-ru.md [hunk 9]
```diff
@@ -0,0 +1,835 @@
...
+```
+
+После этого, добавьте следующие пути к вашему `$PATH` (bash\_profile, zshrc, bashrc, и тп):
+
+```
+export PATH=$PATH:~/.cabal/bin:/opt/cabal/1.22/bin:/opt/ghc/7.8.4/bin:/opt/happy/1.19.5/bin:/opt/alex/3.1.4/bin
+```
+
+*Опционально:* вы можете добавить также `.cabal-sandbox/bin` к вашему пути.
+Код, который вы разрабатываете будет доступен вам из командной строки.
```
Comment (yaroslav) [inline guide-ru.md]: разрабатываете, буден

File: guide-ru.md [hunk 10]
```diff
@@ -0,0 +1,835 @@
...
+```
+
+Поздравляю! Теперь у вас рабочий Haskell!
+
+## Mac OS X
+
+### 10.9
+
+Установите [GHC для Mac OS X](http://ghcformacosx.github.io/) приложение,
+которе включает в себя GHC и Cabal.
```
Comment (yaroslav) [inline guide-ru.md]: которое

File: guide-ru.md [hunk 11]
```diff
@@ -0,0 +1,835 @@
...
+# IO
+
+- [Evaluation order and State tokens](https://www.fpcomplete.com/user/snoyberg/general-haskell/advanced/evaluation-order-and-state-tokens)
+
+- [Unraveling the mystery of the IO monad](http://blog.ezyang.com/2011/05/unraveling-the-mystery-of-the-io-monad/).
+
+- [First class "statements"](http://blog.jle.im/entry/first-class-statements).
+
+- [Haddocks for System.IO.Unsafe.unsafePerformIO](http://hackage.haskell.org/package/base-4.7.0.1/docs/System-IO-Unsafe.html#v:unsafePerformIO)
+  Читая документацию обратите внимание на реализацию unsafeDupablePerformIO
```
Comment (yaroslav) [inline guide-ru.md]: При прочтении, обратите внимание на

File: guide-ru.md [hunk 12]
```diff
@@ -0,0 +1,835 @@
...
+
+## Трансформеры монад
+
+- [A gentle introduction to Monad Transformers](https://github.com/kqr/gists/blob/master/articles/gentle-introduction-monad-transformers.md).
+
+- [Monad transformers step-by-step](http://catamorph.de/documents/Transformers.pdf).
+
+# Тестирование, тесты, спеки, generative/property тестирование
+
+- Это [руководство](https://github.com/kazu-yamamoto/unit-test-example/blob/master/markdown/en/tutorial.md) от Kazu Yamamoto просто фантастичкое.
```
Comment (yaroslav) [inline guide-ru.md]: Yamamoto — просто фантастическое

File: guide-ru.md [hunk 13]
```diff
@@ -0,0 +1,835 @@
...
+
+- [Simple-Conduit](https://github.com/jwiegley/simple-conduit): Хорошая простая
+  библиотека для изучения, как работает стриминг IO вобщем, знания, применимые также
```
Comment (yaroslav) [inline guide-ru.md]: в целом; знания

File: guide-ru.md [hunk 14]
```diff
@@ -0,0 +1,835 @@
...
+
+Взгляните [сюда](https://gist.github.com/bitemyapp/3e6a015760775e0679bf).
+
+## Haddock
+
+1. [Fix your hackage documentation](http://fuuzetsu.co.uk/blog/posts/2014-01-06-Fix-your-Hackage-documentation.html)
+
+2. [Hackage documentation v2](http://fuuzetsu.co.uk/blog/posts/2014-01-06-Hackage-documentation-v2.html)
+
+Заметтье, что эти публикации *слегка устарели*: например, теперь Hackage поддерживает
```
Comment (yaroslav) [inline guide-ru.md]: Заметьте

**Comment by gazay** [comment]
Thank you, @yaroslav!

**Comment by gazay** [comment]
@bitemyapp translation checked and looks like ready for merge. It is similar to German translation, I didn't translate names of articles and talks. Also I'm not clear yet with some of topics in Haskell as I'm beginner. So I hope other russian-speaking haskellers will help with improving this translation later.

**Comment by gazay** [comment]
@bitemyapp I think it's fine to merge already =)

**Comment by bitemyapp** [comment]
@gazay good-o. Thank you! :bear: :ru: :+1: :fireworks: :smile_cat:

**Pull request merged**